.github/workflows/release.yml

```diff
@@ -28,6 +28,10 @@ jobs:
     - name: Nerdbank.GitVersioning
       uses: dotnet/nbgv@v0.4.1
       id: nbgv
+        
+    - name: Update submodule
+      run: |
+        git submodule update --init --recursive --remote 
```

Comment:
```yml
    - name: Checkout
      uses: actions/checkout@v4
      with:
        fetch-depth: 0
        submodules: recursive
```

Reply:
Now that the workflow action is working, do I need to change it?

Reply:
Easy to maintain, also we don't know in future if `git` will be pre-installed on action runners or if it up to date.

Reply:
Well we can change it later.